sdk/formrecognizer/azure-ai-formrecognizer/src/samples/README.md

```diff
@@ -42,16 +42,15 @@ Troubleshooting steps can be found [here][SDK_README_TROUBLESHOOTING].
 See [Next steps][SDK_README_NEXT_STEPS].
 Following section provides code samples for some of the advanced scenarios in Form Recognizer Client library:
 - Get info to help with manually validating the output of the `FormRecognizer` client
-  - [get_manual_validation_info][get_manual_validation_info] and [get_manual_validation_info_async][get_manual_validation_info_async]
+  - [ManualValidationInfo][get_manual_validation_info] and [ManualValidationInfoAsync][get_manual_validation_info_async]
 - See the differences in output when using a custom model trained with labeled data and one trained with unlabeled data
- - [differentiate_custom_forms_with_labeled_and_unlabeled_models][differentiate_custom_forms_with_labeled_and_unlabeled_models] and [differentiate_custom_forms_with_labeled_and_unlabeled_models_async][differentiate_custom_forms_with_labeled_and_unlabeled_models_async]
+ - [DiffCustomFormsLabeledUnlabeledData][differentiate_custom_forms_with_labeled_and_unlabeled_models] and [DiffCustomFormsLabeledUnlabeledDataAsync][differentiate_custom_forms_with_labeled_and_unlabeled_models_async]
```

Comment:
Just to make sure: is it ok that we call the sample DiffCustomFormsLabeledUnlabeledData, but the actual sample file's name has an 'Advanced' prefix

Reply:
Yeah, I was contemplating this. I will update!